trestle/core/markdown/control_markdown_node.py

```diff
@@ -55,10 +86,8 @@ class ControlSectionContent(BaseSectionContent):
     def __init__(self):
         """Initialize section content."""
         super(ControlSectionContent, self).__init__()
-        self.statement_part = None
-        self.objective_part = None
-        self.guidance_part = None
-        self.other_parts = []
+        self.a_part = None
+        self.by_id_name = ''
```

Comment:
Calling it `a_part` is odd to me.  Partly because a statement could have `Part a.` in it.  Is there a better name to describe what it is and what it contains?  Some doc text would help also.

Reply:
`a_part` could represent any part from the markdown. A part is something that starts with two or more hashes. Let me think if there is any other generic enough name for this